baselines/vector_based.py

```diff
@@ -5,10 +5,12 @@
 import shutil
 import tempfile
 
+import bert
```

Comment:
we need a way to make this import work- can we maybe add bert as a git submodule? or is there a pip package?

Reply:
there is a pip package - I added it to the requirements list.